laravel-app/database/seeders/UserPostsTableSeeder.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class UserPostsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        $user_posts = [
+            [
+                'user_id' => 1,
+                'content' => 'The quick brown fox jumps over the lazy dog :O',
+                'created_at' => now(),
+            ],
+            [
+                'user_id' => 1,
+                'content' => 'Time is gold, spend it wisely :)',
+                'created_at' => now(),
+            ],
+        ];
```

Comment:
no need to insert created_at because it's automatically updated once you do insert or create in Eloquent.

Reply:
![image](https://github.com/bpocrafael/microblog-project/assets/144191038/ef58eba0-f9c8-4299-95a8-893243a301cc)
![image](https://github.com/bpocrafael/microblog-project/assets/144191038/b3a99a91-b1c4-4713-9b8f-ef08c4aede16)
![image](https://github.com/bpocrafael/microblog-project/assets/144191038/45462c3f-dc1d-4d19-affa-d0d4f7e703b1)
created_at were not automatically filled upon seeding the db, sir. After removing the created_at from seeders